library/elliptics.h

```diff
@@ -58,28 +58,31 @@ struct dnet_group;
 struct dnet_net_state;
 struct dnet_cmd_stats;
 struct dnet_access_context;
+struct n2_message;
 
 struct dnet_io_req {
-	struct list_head	req_entry;
+	struct list_head		req_entry;
 
-	struct dnet_net_state	*st;
+	struct dnet_net_state		*st;
+	struct n2_message	*n2_msg;
 
-	void			*header;
-	size_t			hsize;
+	// TODO: remove deprecated fields after refactoring complete
+	/*Deprecated*/ void		*header;
```

Comment:
Please, keep original indent
```suggestion
	void		*header; /*Deprecated*/
```